### PR TITLE
Added tlg1181.tlg001.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Aristarchus On the Sizes and Distances of the Sun and the Moon

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7033 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg1181/__cts__.xml`                          | New      |
| `./data/tlg1181/tlg001/__cts__.xml`                   | New      |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1181/__cts__.xml`                          | Passing  |
| `./data/tlg1181/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3682)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7033 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg1181/__cts__.xml`                          | New      |
| `./data/tlg1181/tlg001/__cts__.xml`                   | New      |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1181/__cts__.xml`                          | Passing  |
| `./data/tlg1181/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3685)*

**Comment by AlisonBabeu** [comment]
Hi @sonofmun just took a quick look at this file and for some reason the title has XXXX rather than De magnitudinibus et distantiis solis et lunae.

**Comment by sonofmun** [comment]
Ooops. Nice catch. I forgot to switch out my placeholder.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7033 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg1181/__cts__.xml`                          | New      |
| `./data/tlg1181/tlg001/__cts__.xml`                   | New      |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1181/__cts__.xml`                          | Passing  |
| `./data/tlg1181/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3692)*

File: data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,1016 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+ <title>De magnitudinibus et distantiis solis et lunae</title>
```
Comment (planatheisa) [inline data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml]: Missing lang-tag.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7033 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg1181/__cts__.xml`                          | New      |
| `./data/tlg1181/tlg001/__cts__.xml`                   | New      |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1181/__cts__.xml`                          | Passing  |
| `./data/tlg1181/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg1181/tlg001/tlg1181.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3780)*

**Pull request merged**